rust/config.toml

```diff
@@ -0,0 +1,419 @@
+# Sample TOML configuration file for building Rust.
```

Comment:
Nit: Can we `wget` this sample file from somewhere? I'd prefer it if the Rust Dockerfile downloads a recent version of this sample file, instead of importing it into this repository (and promising to always keep it in sync manually). Unless we need to fork it somehow.

Reply:
The sample file serves as a documentation, and we comment out lines to enable options we need. Thus, i think including in-tree is the best solution. I will remove the "this is a sample" line though.